### PR TITLE
Installation Instructions: Adding Pulumi Homebrew Tap

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

**Review by susanev — APPROVED** [review]

File: themes/default/content/docs/get-started/install/_index.md [hunk 1]
```diff
@@ -43,6 +45,15 @@ Subsequent updates can be installed in the usual way:
 $ brew upgrade pulumi
 ```
 
+### Community Homebrew
+
+A Pulumi formula is available on the Community Homebrew. If you do not have the Pulumi tap installed, then you can
```
Comment (Frassle) [inline themes/default/content/docs/get-started/install/_index.md:50]: Link to the community formula on github?

File: themes/default/content/docs/get-started/install/_index.md [hunk 2]
```diff
@@ -27,12 +27,14 @@ This page contains detailed instructions for [installing Pulumi](#installing-pul
...
-### Homebrew
+### Official Pulumi Homebrew Tap
 
-You can install Pulumi through the [Homebrew package manager](https://brew.sh/):
+You can install Pulumi through the [Homebrew package manager](https://brew.sh/) and using our official
+[Pulumi Homebrew Tap](https://github.com/pulumi/homebrew-tap/)
 
 ```bash
-$ brew install pulumi
+$ brew tap pulumi/tap
```
Comment (lukehoban) [inline themes/default/content/docs/get-started/install/_index.md:36]: Is this line needed?

I noticed another project I used recently didn't include this as a separate step, and it worked anyway:

https://nitric.io/docs/installation

**Review by lukehoban — APPROVED** [review]
LGTM - though one question about whether we can simplify even further.

File: themes/default/content/docs/get-started/install/_index.md (hunk 2 above)
Comment (stack72) [reply themes/default/content/docs/get-started/install/_index.md]: Good call - we don’t technically need it

File: themes/default/content/docs/get-started/install/_index.md (hunk 1 above)
Comment (stack72) [reply themes/default/content/docs/get-started/install/_index.md:49]: That'll go in the accompanying blog post - I don't want a CTA that takes people away from the install :)

**Pull request merged**